### PR TITLE
feat: adds admin reference and updated some links

### PR DESCRIPTION
**Pull request by killme2008**
## What's Changed in this PR

*Describe the change in this PR*

* Fixed administration function statements, use `admin` instead of `select`.
* Adds `ADMIN` statement reference.
* Adds benchmark reports.
* Fixed some links.

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [x] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/reference/sql/functions/overview.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Where the `datatype` can be any valid Arrow data type in this [list](https://arr
 DataFusion [String Function](./df-functions#string-functions).GreptimeDB provides:
 * `matches(expression, pattern)` for full text search. 
 
-<!-- TODO: link to full-text searching user guide. -->
+For details, read the [Query Logs](user-guide/logs/query-logs.md).
```
Comment (nicecui) [inline docs/reference/sql/functions/overview.md:42]:
```suggestion
For details, read the [Query Logs](/user-guide/logs/query-logs.md).
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/sql/functions/overview.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ arrow_cast(expression, datatype)
 DataFusion [字符串函数](./df-functions#string-functions)。GreptimeDB 提供：
 * `matches(expression, pattern)` 用于全文检索。
 
-TODO：链接到全文检索用户指南。
+阅读[查询日志](user-guide/logs/query-logs.md)文档获取更多详情。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/sql/functions/overview.md:41]:
```suggestion
阅读[查询日志](/user-guide/logs/query-logs.md)文档获取更多详情。
```

**Review by nicecui — COMMENTED** [review]
LTGM

**Review by nicecui — APPROVED** [review]

**Comment by sunng87** [comment]
I feel we need to use a verb for an action in SQL, like select/delete/update are all verbs that followed. `admin` are mostly used as a noun in computer. Although it has a verb form "administer", it [means someone is in charge of something](https://www.vocabulary.com/dictionary/administer).

My suggestion is to use another keyword for this type of function execution, like

- EXECUTE function
- SELECT ADMIN function (admin becomes adj. ADMINISTRATIVE, similar to SELECT DISTINCT)

**Comment by killme2008** [comment]
It's the statement of clickhouse. I think it's good. We already discussed it.

**Comment by sunng87** [comment]
https://clickhouse.com/docs/en/sql-reference/statements

I didn't see similar statement from clickhouse's docs.

**Comment by killme2008** [comment]
> https://clickhouse.com/docs/en/sql-reference/statements
> 
> I didn't see similar statement from clickhouse's docs.

I don't think it's a matter of this keyword. 

It's proposed by @evenyag at https://github.com/GreptimeTeam/greptimedb/issues/4462

**Comment by killme2008** [comment]
> https://clickhouse.com/docs/en/sql-reference/statements
> 
> I didn't see similar statement from clickhouse's docs.

My mistake, just like the `system` statement:

https://clickhouse.com/docs/en/sql-reference/statements/system

**Review by fengjiachun — APPROVED** [review]
LGTM

**Pull request merged**